integration-cli/docker_cli_pull_local_test.go

```diff
@@ -702,49 +702,40 @@ func (s *DockerRegistriesSuite) TestPullWithPolicy(c *check.C) {
 	dgst := matches[1]
 
 	// test that I can pull by a digest and tag when the signature is valid for the tag
-	// default referencesByDigest == allow
-	//
-	// FIXME: requires containers/image#129 to make this test below work
-	//        when that's in, change this test to assert success
-	//
 	dgstRef := fmt.Sprintf("%s/busybox@%s", s.reg1.url, dgst)
-	_, _, err = dockerCmdWithError("pull", dgstRef)
-	c.Assert(err, check.NotNil)
+	dockerCmd(c, "pull", dgstRef)
 
 	dgstRefUnsigned := fmt.Sprintf("%s/busybox@%s", s.reg1.url, unsignedDigest)
 	out, _, err = dockerCmdWithError("pull", dgstRefUnsigned)
 	c.Assert(err, check.NotNil)
 	c.Assert(out, checker.Contains, "no signature exists")
 
 	// test that I can pull by tag but NOT by digest when the signature is valid for the tag
```

Comment:
AFAICT there is only https://github.com/projectatomic/docker/pull/213/files#diff-8c285cfed55b08d027aed4e479cb9435R736 testing digest use.